web/ui/module/codemirror-promql/src/complete/hybrid.ts

```diff
@@ -208,10 +215,28 @@ export function computeStartCompletePosition(state: EditorState, node: SyntaxNod
   return start;
 }
 
+function isItATopKLikeAggregationFunc(functionCallBody: SyntaxNode): boolean {
+  const prevSibling = functionCallBody.prevSibling;
+  if (prevSibling !== null && prevSibling.type.id === AggregateOp) {
+    const aggregationOpType = prevSibling.firstChild;
+    if (
+      aggregationOpType !== null &&
+      (aggregationOpType.type.id == Topk ||
+        aggregationOpType.type.id === Bottomk ||
+        aggregationOpType.type.id === LimitK ||
+        aggregationOpType.type.id === LimitRatio ||
+        aggregationOpType.type.id === CountValues)
+    ) {
```

Comment:
Do you prefer this over just `[Topk, Bottomk, ...].includes(aggregationOpType.type.id)`?

Reply:
yeah I prefer that. Thanks !